source/extensions/omni.isaac.lab/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 ---------
 
+0.27.12 (2024-11-02)
+~~~~~~~~~~~~~~~~~~~
+Fixed
+^^^^^
+* Fixed the correct projection types of fisheye camera in :class:`omni.isaac.lab.sim.spawners.sensors.sensors_cfg.FisheyeCameraCfg`.
+
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~

Fixed
^^^^^

* Corrected the projection types of fisheye camera in :class:`omni.isaac.lab.sim.spawners.sensors.sensors_cfg.FisheyeCameraCfg`.
  Earlier, the projection names used snakecase instead of camelcase.

```